new smart contracts/contracts/Collection.sol

```diff
@@ -3,36 +3,91 @@
 
 pragma solidity ^0.8.13;
 
-import {ERC721A} from "erc721a/contracts/ERC721A.sol";
-import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
-import {FragnovaBaseUri} from "./FragnovaBaseUri.sol";
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import {OwnableUpgradeable} from "./OwnableUpgradeable.sol";
```

Comment:
why OwnableUpgradeable and not our Ownable.sol?

Reply:
1. Because it's best practice for `Upgradeable` contracts to not even have a `constructor`
2. Because your `Ownable.sol` contract wasn't tested and also I was confused what `DEFAULT_PROTOCOL_OWNER ` is 😅

Reply:
But who said we wanted it upgradeable?

Reply:
It's not upgradeable. OpenZepplin's naming isn't on the ball here. They should have named it `OwnableInitializable` lol